pkg/configutil/testdata/valid_signingkeys/signingkeys.json

```diff
@@ -0,0 +1,10 @@
+{
+    "default": "e2e",
+    "keys": [
+        {
+            "name": "e2e",
+            "keyPath": "notation/localkeys/e2e.key",
+            "certPath": "notation/localkeys/e2e.crt"
+        }
+    ]
+}
```

Comment:
We no longer need this one because `LoadSigningkeysOnce` is removed as part of https://github.com/notaryproject/notation/pull/529

Reply:
We need it for testing TestResolveKey()